### PR TITLE
typo: fix some typo

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?

Issue Number: Ref #5892

### What is changed and how does it work?

```commit-message
fix some typos
```

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: server/config/config.go [hunk 1]
```diff
@@ -563,7 +563,7 @@ type ScheduleConfig struct {
 	// SplitMergeInterval is the minimum interval time to permit merge after split.
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval" json:"split-merge-interval"`
 	// SwitchWitnessInterval is the minimum interval that allows a peer to become a witness again after it is promoted to non-witness.
-	SwitchWitnessInterval typeutil.Duration `toml:"switch-witness-interval" json:"swtich-witness-interval"`
```
Comment (rleungx) [inline server/config/config.go:566]: Will it have a compatibility issue?

Comment (HuSharp) [reply server/config/config.go:566]: This is a feature that has no GA and is only used within pd...
But I think it is possible to meet a compatibility issue

**Review by nolouch — APPROVED** [review]

File: server/config/config.go (hunk 1 above)
Comment (nolouch) [reply server/config/config.go:566]: not GA we can fix it directly.

**Pull request merged**